modules/core/bin/install-deps/debian.sh

```diff
@@ -101,7 +101,7 @@ if [ -n "$APT_DEPS" ]; then
     sudo apt update
     sudo apt install -y $APT_DEPS;
     if [ -n "$INSTALLED_CLANGD" ]; then
-        sudo update-alternatives --install /usr/bin/clangd clangd /usr/bin/clangd-17 100
+        sudo update-alternatives --install /usr/bin/clangd clangd /usr/bin/clangd-12 100
```

Comment:
clangd-12 is incredibly old and likely broken for modern CUDA. clang-format-12 will also most likely regress from clang-format-17, which is not ideal. Is there a reason we're downgrading these?

Reply:
Good catch, this was part of my old tests, I updated it to `clang 18` (which is compatible with `ubuntu 24.04`)